src/workerd/jsg/util.h

```diff
@@ -486,4 +522,29 @@ using WontImplement = Unimplemented;
 kj::Maybe<kj::String> checkNodeSpecifier(kj::StringPtr specifier);
 bool isNodeJsCompatEnabled(jsg::Lock& js);
 
+// ======================================================================================
+// Fast API
+
+namespace fastApi {
+struct CallCounter {
+  uint32_t slow = 0;
+  uint32_t fast = 0;
+
+  void reset() {
+    slow = 0;
+    fast = 0;
+  }
+
+  bool operator==(const kj::Tuple<uint32_t, uint32_t>& rhs) {
```

Comment:
`kj::Tuple` should almost never be used. The only place `kj::Tuple` actually make sense is when dealing with vararg templates. Anywhere else, it's better to just use a struct.

In fact, if you instead changed this equality operator to take `const CallCounter&` as the rhs argument, then instead of writing:

```
KJ_ASSERT(jsg::fastApi::callCounter == kj::tuple(1, 0));
```

You could write:

```
KJ_ASSERT(jsg::fastApi::callCounter == {1, 0});
```

It's actually less code!

Reply:
Ah yes, that's definitely less code! Thanks @kentonv